Sources_v3/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -115,7 +126,7 @@ public class _ChatChannelListController<ExtraData: ExtraDataTypes>: DataControll
     }
     
     /// Initializing of `channelListObserver` will start local data observing.
-    /// In most cases it will be done by accesing `channels` but it's possible that only
+    /// In most cases it will be done by accusing `channels` but it's possible that only
```

Comment:
I totally did not understand what this sentence means

Reply:
Fixed in 8a8b937d4410137fa5e244b68b9441caa452345e